CONTRIBUTING.md

```diff
@@ -108,7 +108,7 @@ Or:
 
 After you've bootstrapped the repo, you would probably want to work on individual packages.
 
-All packages in the repo have a two useful scripts: `prepare` and `watch`. In order to execute
```

Comment:
"prepare" was actually renamed to "build", but I wouldn't remove it

Reply:
I'm a little confused when I run `prepare` in `@aws-ckd/aws-ec2` I get:

```
lr prepare
info cli using local version of lerna
lerna notice cli v3.4.1
lerna info filter [ '@aws-cdk/aws-ec2' ]
lerna success run No packages found with the lifecycle script 'prepare'
```

So what would a new contributor do with `prepare`?